clouds/aws/create_aws_network_resources.py

```diff
@@ -0,0 +1,793 @@
+import json
+import os
+from ipaddress import IPv4Network
+
+import click
+from aws_utils import verify_aws_config, verify_aws_credentials
+from ocp_resources.utils import TimeoutExpiredError, TimeoutSampler
+from simple_logger.logger import get_logger
+
+from clouds.aws.session_clients import ec2_client as test_client
```

Comment:
why import as? `as test_client` 
you take a self-descriptive name and make it as non-descriptive one

Reply:
leftover, removed it